rosa_planning/rosa-cloud-expert-prereq-checklist.adoc

```diff
@@ -157,6 +157,16 @@ Prerequisites needed from a networking standpoint.
 
 * Configure your firewall to allow access to the domains and ports listed in xref:../rosa_planning/rosa-sts-aws-prereqs.adoc#osd-aws-privatelink-firewall-prerequisites_rosa-sts-aws-prereqs[AWS firewall prerequisites].
 
+=== Additional custom security groups
+
+When you create a cluster using an existing VPC, you have the option to add additional custom security groups during cluster creation. Complete these prerequisites before you create the cluster:
+
+* Create the custom security groups in AWS before you create the cluster.
+* Associate the custom security groups with the VPC that you are using to create the cluster. Do not associate the custom security groups with any other VPC.
+* You may need to request additional AWS quota for `Security groups per network interface`.
+
+See the requirements for link:https://access.redhat.com/documentation/en-us/red_hat_openshift_service_on_aws/4/html/prepare_your_environment/rosa-sts-aws-prereqs#rosa-security-groups_rosa-sts-aws-prereqs[Security groups] for more details.
```

Comment:
For this hard link, it is in an assembly, so you can actually use an xref here. That would be preferable so that if someone is viewing these docs from docs.openshift.com, they aren't redirected suddenly to the portal.

Reply:
Corrected to `xref:../../rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-aws-prereqs.adoc#rosa-security-groups_rosa-sts-aws-prereqs[Security groups]` which is hopefully correct.